tests/app/forms/validation/test_number_range_validator.py

```diff
@@ -2,10 +2,12 @@
 from unittest.mock import Mock, patch
 from wtforms.validators import ValidationError
 
+from app.forms.field_handlers.number_handler import NumberHandler
+from app.forms.fields.integer_field_with_separator import IntegerFieldWithSeparator
+from app.forms.field_factory import get_field_handler
```

Comment:
Testing the validators shouldn't require the field or handler, and we shouldn't have any schema in these tests. If we're testing that a validator is attached to a field, then that test should live in the handler tests.